DQM/L1TMonitor/src/L1TStage2CaloLayer2.cc

```diff
@@ -293,9 +296,17 @@ void L1TStage2CaloLayer2::analyze(const edm::Event & e, const edm::EventSetup &
 	  stage2CaloLayer2ETTEMRank_->Fill(itEtSum->hwPt());
 	} else if (l1t::EtSum::EtSumType::kTowerCount == itEtSum->getType()) {
 	  stage2CaloLayer2TowCount_->Fill(itEtSum->hwPt());
-	} else{
+	} else if (l1t::EtSum::EtSumType::kTotalHt == itEtSum->getType()){
```

Comment:
@vukasinmilosevic the differences observed in the DQM plots for non HI workflows come from here I guess, am I correct? Is this desired?

Reply:
Hi, that is the only difference in the code (besides the additional plots). I was told to use that EtSumType (kTotalHT) connected to HTTRank